lib/faker/default/company.rb

```diff
@@ -5,57 +5,163 @@ class Company < Base
 
-      # Generate a buzzword-laden catch phrase.
+      ##
+      # Produces a company catch phrase.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.catch_phrase #=> "Grass-roots grid-enabled portal"
+      #
+      # @faker.version 1.6.0
       def catch_phrase
         translate('faker.company.buzzwords').collect { |list| sample(list) }.join(' ')
       end
 
+      ##
+      # Produces a company buzzword.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.buzzword #=> "flexibility"
+      #
+      # @faker.version 1.8.7
       def buzzword
         sample(translate('faker.company.buzzwords').flatten)
       end
 
+      ##
+      # Produces some company BS.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.bs #=> "empower customized functionalities"
+      #
+      # @faker.version 1.6.0
       # When a straight answer won't do, BS to the rescue!
       def bs
         translate('faker.company.bs').collect { |list| sample(list) }.join(' ')
       end
 
+      ##
+      # Produces a company EIN (Employer Identification Number).
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.ein #=> "07-4009024"
+      #
+      # @faker.version 1.6.0
       def ein
         format('%09d', rand(10**9)).gsub(/(\d{2})(\d{7})/, '\\1-\\2')
       end
 
+      ##
+      # Produces a company duns number.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.duns_number #=> "70-655-5105"
+      #
+      # @faker.version 1.6.0
       def duns_number
         format('%09d', rand(10**9)).gsub(/(\d{2})(\d{3})(\d{4})/, '\\1-\\2-\\3')
       end
 
+      ##
+      # Produces a company logo.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.logo #=> "https://pigment.github.io/fake-logos/logos/medium/color/12.png"
+      #
+      # @faker.version 1.8.7
       # Get a random company logo url in PNG format.
       def logo
         rand_num = rand(1..13)
         "https://pigment.github.io/fake-logos/logos/medium/color/#{rand_num}.png"
       end
 
+      ##
+      # Produces a company type.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.type #=> "Partnership"
+      #
+      # @faker.version 1.8.7
       def type
         fetch('company.type')
       end
 
+      ##
+      # Produces a company profession.
+      #
+      # @return [String]
+      #
+      # @example
+      #   Faker::Company.profession #=> "factory worker"
+      #
+      # @faker.version 1.6.0
       def profession
         fetch('company.profession')
       end
 
-      # rubocop:disable Style/AsciiComments
-      # Get a random Spanish organization number. See more here https://es.wikipedia.org/wiki/Número_de_identificación_fiscal
-      # rubocop:enable Style/AsciiComments
+      ##
+      # Produces a company spanish_organisation_number.
```

Comment:
You missed this one 😅 

```suggestion
      # Produces a company spanish organisation number.
```